src/graph_compute/phase_scheduling.h

```diff
@@ -40,6 +40,8 @@ static inline bool interChunkDependency(vid_t v, vid_t w) {
   static const vid_t chunkMask = (1 << CHUNK_BITS) - 1;
   if ((v >> CHUNK_BITS) == (w >> CHUNK_BITS)) {
     return false;
+  } else if ((v & chunkMask) == (w & chunkMask)) {
```

Comment:
It returns true if v is at a lower position within its chunk than w is within its chunk.  If they are at the same position, within their respective chunks, we break ties by the chunk number.